test/features/install_managementconsole.feature

```diff
@@ -0,0 +1,18 @@
+@managementconsole
+Feature: Install Kogito Management Console
+
+  Background:
+    Given Namespace is created
+    And Kogito Operator is deployed with Infinispan and Kafka operators
+
+  Scenario Outline: Install Kogito Management Console
+    Given "<installer>" install Kogito Data Index with 1 replicas
```

Comment:
since the merge of https://github.com/kiegroup/kogito-cloud-operator/pull/213,  `"<installer>" install Kogito [...]` is replaced directly by `Install Kogito [...]`
and the CLI/CR decision is made directly into the code.
Check steps in `install_dataindex.feature` for example

Reply:
Fixed.